package.json

```diff
@@ -1,16 +1,17 @@
 {
   "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "android": "expo start --android",
-    "eject": "expo eject",
-    "ios": "expo start --ios",
-    "lint": "./node_modules/.bin/eslint __tests__ src --ext .js,.jsx,.ts,.tsx",
     "start": "expo start",
+    "start:android": "expo start --android",
+    "start:ios": "expo start --ios",
+    "build:android": "eas build --platform android",
+    "build:ios": "eas build --platform ios",
+    "submit:android": "eas submit --platform android",
+    "submit:ios": "eas submit -platform ios",
```

Comment:
maybe we should combine build and submit here, because submit always requires a prior build. or we always do `yarn build:ios && yarn submit:ios` to combine them.

Reply:
i think we will need multiple commands anyways, so let's do not have a combined one.

Reply:
there is an `--auto-submit` flag for the `build` command, which seems to combine build and submit. This could be interesting for our `yarn submit:ios` as we always need to build before submission.

Reply:
on the other hand regarding CI functionalities in a future release step, a build cannot happen on the server due to necessary cli prompts. so we still need a submit only here i guess.